docs/features/writing-different-telemetry-types.md

```diff
@@ -30,10 +30,49 @@ Dependencies allow you to track how your external dependencies are doing to give
 
 We provide support for the following dependencies:
 
+- [Azure Service Bus](#measuring-azure-service-bus-dependencies)
 - [Custom](#measuring-custom-dependencies)
 - [HTTP](#measuring-http-dependencies)
 - [SQL](#measuring-sql-dependencies)
 
+### Measuring Azure Service Bus dependencies
+
+Here is how you can report an Azure Service Bus dependency:
```

Comment:
Maybe it's better here to mention what we support instead of diving into an example immediately?

For example:
```suggestion

We allow you to measure Azure Service Bus dependencies for both queues & topics.

Here is how you can report an Azure Service Bus Queue dependency:
```

Reply:
Yes, and also unknown entities.